src/VirtualClient/VirtualClient.Main/profiles/PERF-BLENDER-AMD.json

```diff
@@ -0,0 +1,65 @@
+{
+  "Description": "Blender GPU/CPU Performance Workload",
+  "Metadata": {
+    "RecommendedMinimumExecutionTime": "00:10:00",
+    "SupportedPlatforms": "win-x64",
+    "SupportedOperatingSystems": "Windows"
+  },
+  "Parameters": {
+    "GpuModel": "",
+    "BlenderVersion": "3.6.0",
+    "Scenes": "monster junkshop classroom",
+    "DeviceTypes": "CPU, HIP"
+  },
+  "Actions": [
+    {
+      "Type": "BlenderExecutor",
+      "Parameters": {
+        "Scenario": "MeasureGPU&CPUPerformance",
```

Comment:
$ ampersand is less preferred than 'And' because escaping and also delimiting but your choice